terraform/envs/gcp/main.tf

```diff
@@ -81,3 +81,13 @@ module "coder-server" {
   oidc_client_id       = var.coder_auth == "oidc" ? var.oidc_client_id : null
   oidc_client_secret   = var.coder_auth == "oidc" ? var.oidc_client_secret : null
 }
+
+module "neo4j" {
+  source               = "github.com/aisingapore/kapitan-hull-admin//terraform/modules/neo4j?ref=neo4j-integration"
```

Comment:
If we're pulling directly from github, best to set to a shallow clone, i.e. `depth=1`

Reply:
Added in [dac817a](https://github.com/aisingapore/kapitan-hull-admin/pull/15/commits/dac817a214dacb7290287c642adb85fccbcb6b9d)